app/assets/javascripts/tylium/modules/qa_visibility.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener('turbolinks:load', function () {
```

Comment:
`qa-viewer` seems like it's only present in the QA page. If that's the case, can we make this a QA specific JS and follow the naming convention? Something like `tylium/pages/qa/table.js`? Also instead of `qa-viewer` does it make sense to use `body.qa` instead?